packages/gen-ai-hub/src/orchestration/orchestration-types.ts

```diff
@@ -1,40 +1,40 @@
 import {
   ChatMessages,
   FilteringModuleConfig,
-  LLMModuleConfig
+  LLMModuleConfig,
+  TemplatingModuleConfig
 } from './client/api/index.js';
 
 /**
- * Wrapper object to configure prompt.
+ * Prompt configuration.
  */
-export interface PromptConfig {
+export interface Prompt {
   /**
    * History.
    */
-  messages_history?: ChatMessages;
-  /**
-   * Template.
-   */
-  template: ChatMessages;
+  messagesHistory?: ChatMessages;
+
   /**
-   * Template Parameters.
+   * Template parameters.
    */
-  template_params?: Record<string, string>;
+  inputParams?: Record<string, string>;
 }
 
+// TODO: why do we have this extra type? and if there is a reason, why does it not apply to the filtering module?
 /**
- * Wrapper object to configure LLMModule.
+ * LLMModule configuration.
  */
 export type LlmConfig = LLMModuleConfig;
 
 /**
- * Wrapper object to encompass Orchestration options.
+ * Orchestration module configuration.
  */
-export interface OrchestrationCompletionParameters {
+export interface OrchestrationModuleConfig {
+  // TODO: remove "config" for all the configs below
```

Comment:
[req] The type names still need to be adjusted.

Reply:
Since the types are generated it probably makes no sense to adjust the original name, I suppose "import as ..." should be used here?

Reply:
The idea is to also export the resultant renamed types